modules/sdk-unified-wallet/package.json

```diff
@@ -14,10 +14,12 @@
   },
   "dependencies": {
     "@bitgo/sdk-core": "^5.2.0",
-    "@bitgo/statics": "^14.1.0"
+    "@bitgo/statics": "^14.1.0",
+    "bitgo": "^16.4.0",
```

Comment:
please remove. Instead write your unit tests with TestBitGo and BitGoAPI. See other coin module unit tests for example on how to do this, but adding this dependency is going to 1000x your bundle size because now you depend on every other coin in the repo.